src/collections/_documentation/performance/discover/query-builder.md

```diff
@@ -314,7 +314,24 @@ A down arrow indicates sorting the column in descending order. An up arrow indic
 
 ### Download CSV
 
-Note, only the current values visible on the table are exported, not the paginated results. To export the table as CSV, click on the "Download CSV" button.
+If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while, so sit back and relax. You'll get an email with the download link once they're ready.
```

Comment:
:) Let's be more specific about how long this will take. Do we have an estimation of time?

Reply:
Also, our docs have a more formal tone. Let's remove the informal voice here.
```suggestion
If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while. You'll get an email with the download link once they're ready.
```

Reply:
Unfortunately this can be anywhere from 1 second to over an hour, so I don't have a good estimate to put here.